client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -79,7 +93,7 @@
                 @click="abortEdit()">
                 <dp-icon
                   icon="xmark"
-                  aria-hidden />
+                  aria-hidden="true" />
```

Comment:
I thought, its recommended to drop setting the inherit value.

Reply:
I don't think that's true? Also, PHPStorm complains about it: 
> aria-hidden requires value

Reply:
ok. Strange. I was sure phpstorm told me about not setting true if its the default. But maybe it was for other attributes and it depends. I love it..